lib/countly.js

```diff
@@ -219,10 +224,18 @@
                 if (Countly.q.constructor !== Array) {
                     Countly.q = [];
                 }
+                
                 heartBeat();
                 store("cly_id", Countly.device_id);
                 var i = 0;
                 Countly.utm = Countly.utm || {"source":true, "medium":true, "campaign":true, "term":true, "content":true};
+                
+                // if test mode enabled; define sample values
+                if (Countly.test_mode) {
+                    store('cly_cmp_id', 'test_campaign_id');
+                    store('cly_cmp_uid', 'test_campaign_uid');
+                }
```

Comment:
Don't need this too, let's try to make lib as small as possible :)
So these values can be passed by URL parameters when you load it in casperjs like `test.html?cly_cmp_id=test_campaign_id&cly_cmp_ui=test_campaign_uid`